controls/srg_ctr/SRG-APP-000456-CTR-001125.yml

```diff
@@ -5,4 +5,8 @@ controls:
   title: '{{{ full_name }}} registry must contain the latest images with most
     recent updates and execute within the container platform runtime as authorized
     by IAVM, CTOs, DTMs, and STIGs. '
-  status: pending
+  status: automated
+  rules:
+    - imagestream_sets_schedule
+    - ocp_allowed_registries
+    - ocp_allowed_registries_for_import
```

Comment:
Pleasure ensure end this file with a newline. The check eof script will be updated to catch this in the future.

Reply:
thanks for catching this and even more for working on automation